Node 22 broke master

Updating to Node 22 (#1337) broke some end-to-end tests.

## OIDC e2e tests

https://github.com/getodk/central-backend/actions/runs/12247137794/job/34164411356

```
2024-12-10T00:52:32.2249294Z [1A[2K  1 failed
2024-12-10T00:52:32.2250377Z     [webkit] › oidc-login.spec.js:31:5 › happy › can log in (no next param) ────────────────────────
2024-12-10T00:52:32.2251085Z   20 passed (1.0m)
2024-12-10T00:52:32.2251685Z [HPM] server close signal received: closing proxy server
```

## s3 e2e tests

These appear to have been broken because of an assertion on the stderr output of CLI commands.  `stderr` output has changed due to node emitting new deprecation warnings, e.g.:
```
(node:999316) [DEP0044] DeprecationWarning: The `util.isArray` API is deprecated. Please use `Array.isArray()` instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:999316) [DEP0055] DeprecationWarning: The `util.isRegExp` API is deprecated. Please use `arg instanceof RegExp` instead.
(node:999316) [DEP0047] DeprecationWarning: The `util.isDate` API is deprecated.  Please use `arg instanceof Date` instead.
```

https://github.com/getodk/central-backend/actions/runs/12247137790/job/34164411187

```
 1) s3 support
       should handle s3 connection failing:
     AssertionError: expected 'Command failed: exec node lib/bin/s3 upload-pending\n(node:4522) [DEP0044] DeprecationWarning: The `util.isArray` API is deprecated. Please use `Array.isArray()` instead.\n(Use `node --trace-deprecation ...` to show where the warning was created)\n(node:4522) [DEP0055] DeprecationWarning: The `util.isRegExp` API is deprecated. Please use `arg instanceof RegExp` instead.\n(node:4522) [DEP0047] DeprecationWarning: The `util.isDate` API is deprecated.  Please use `arg instanceof Date` instead.\nAggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1128:18)\n    at afterConnectMultiple (node:net:1693:7) {\n  code: \'ECONNREFUSED\',\n  [errors]: [\n    Error: connect ECONNREFUSED ::1:9000\n        at createConnectionError (node:net:1656:14)\n        at afterConnectMultiple (node:net:1686:16) {\n      errno: -111,\n      code: \'ECONNREFUSED\',\n      syscall: \'connect\',\n      address: \'::1\',\n      port: 9000\n    },\n    Error: connect ECONNREFUSED 127.0.0.1:9000\n        at createConnectionError (node:net:1656:14)\n        at afterConnectMultiple (node:net:1686:16) {\n      errno: -111,\n      code: \'ECONNREFUSED\',\n      syscall: \'connect\',\n      address: \'127.0.0.1\',\n      port: 9000\n    }\n  ]\n}node:internal/process/promises:394\n    triggerUncaughtException(err, true /* fromPromise */);\n    ^\n\nAggregateError [ECONNREFUSED]: \n    at internalConnectMultiple (node:net:1128:18)\n    at afterConnectMultiple (node:net:1693:7) {\n  code: \'ECONNREFUSED\',\n  [errors]: [\n    Error: connect ECONNREFUSED ::1:9000\n        at createConnectionError (node:net:1656:14)\n        at afterConnectMultiple (node:net:1686:16) {\n      errno: -111,\n      code: \'ECONNREFUSED\',\n      syscall: \'connect\',\n      address: \'::1\',\n      port: 9000\n    },\n    Error: connect ECONNREFUSED 127.0.0.1:9000\n        at createConnectionError (node:net:1656:14)\n        at afterConnectMultiple (node:net:1686:16) {\n      errno: -111,\n      code: \'ECONNREFUSED\',\n      syscall: \'connect\',\n      address: \'127.0.0.1\',\n      port: 9000\n    }\n  ]\n}\n\nNode.js v22.12.0\n' to match /Command failed: exec node lib\/bin\/s3 upload-pending\n(AggregateError.*)?Error: (connect ECONNREFUSED|read ECONNRESET|socket hang up|write EPIPE)/s
      at Assertion.fail (/home/runner/work/central-backend/central-backend/node_modules/should/cjs/should.js:275:17)
      at Assertion.value [as match] (/home/runner/work/central-backend/central-backend/node_modules/should/cjs/should.js:356:19)
      at expectRejectionFrom (test.js:441:46)
      at async Context.<anonymous> (test.js:250:5)
```

--- a/test/e2e/s3/test.js
+++ b/test/e2e/s3/test.js
@@ -247,11 +247,7 @@ describe('s3 support', () => {
     // then
     // N.B. These errors are as observed, and demonstrate that the root error is shared
     // with the user.  They are not something to try to retain if implementation changes.
-    await expectRejectionFrom(uploading, new RegExp(
-      'Command failed: exec node lib/bin/s3 upload-pending\n' +
-          '(AggregateError.*)?Error: (connect ECONNREFUSED|read ECONNRESET|socket hang up|write EPIPE)',
-      's',
-    ));
+    await expectRejectionFrom(uploading, /(AggregateError.*)?Error: (connect ECONNREFUSED|read ECONNRESET|socket hang up|write EPIPE)/);
     // and
     await assertNewStatuses({ uploaded: 1, failed: 1 });
   });

